docs/WritingTests.asciidoc

```diff
@@ -219,3 +219,151 @@ $ /usr/share/openqa/script/clone_job.pl --from https://openqa.opensuse.org  --ho
 
 Use qemu-img snapshot -l something.img to find out what snapshots
 are in the image.
+
+Writing multi-machine tests
+---------------------------
+[id="mm-tests"]
+
+Scenarios requiring more than one system under test (SUT), like High Availability testing, are covered as multi-machine tests (MM tests) in section.
+
+OpenQA approaches this problem by assigning dependencies between individual jobs. This means following:
+
+* _everything needed for MM tests must be running as a test job_ (or you are on your own), even support infrastructure (custom DHCP, NFS,
+etc. if required), which in principle is not part of the actual testing, must have a defined test suite so a test job can be created
+* OpenQA scheduler makes sure _tests are started as a group_ and in right order, _canceled as a group_ if some dependencies are violated and _cloned as
```

Comment:
hmm. while we use cancelled everywhere in the code, it seems to be british english?